### PR TITLE
[docker-sonic-mgmt] Add IxNetwork python client 

### PR DESCRIPTION
**Pull request by baiwei0427**
Signed-off-by: Wei Bai baiwei0427@gmail.com

**- Why I did it**
Add [IxNetwork python client](https://pypi.org/project/ixnetwork-restpy/) to sonic-mgmt docker 

**- How I did it**
Modify Dockerfile of sonic-mgmt docker

**- How to verify it**
Build the new sonic-mgmt docker. Try importing ixnetwork_restpy inside the docker. For example:
```
username@d73f7075fd2f:~$ python
Python 2.7.12 (default, Apr 15 2020, 17:07:12)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import ixnetwork_restpy
>>>
```

**- Description for the changelog**

Add IxNetwork python client to sonic-mgmt docker

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-sonic-mgmt/Dockerfile.j2 [hunk 1]
```diff
@@ -74,6 +74,7 @@ RUN pip install ipaddr \
                 rpyc \
                 tabulate \
                 textfsm \
+                ixnetwork-restpy \
```
Comment (lguohan) [inline dockers/docker-sonic-mgmt/Dockerfile.j2:77]: can you specify the version?

**Comment by baiwei0427** [comment]
retest this please

**Comment by baiwei0427** [comment]
retest vsimage please

**Comment by baiwei0427** [comment]
retest vsimage please

**Pull request merged**